static/src/javascripts/lib/config.spec.js

```diff
@@ -0,0 +1,60 @@
+// @flow
+/* eslint-disable guardian-frontend/global-config */
+
+const prevConfig = window.guardian.config;
+Object.assign(window.guardian.config, {
+    page: {
+        tones: 'foo',
+        series: 'bar',
+        references: [{ baz: 'one' }, { baz: 'two' }],
+        webPublicationDate: '2013-03-20T17:07:00.000Z',
+        pageId: 'politics/2017/mar/14/ukip-donor-arron-banks-says-he-has-quit-party-to-set-up-ukip-20',
+    },
+});
+
+// We need to hoist setting the window.guardian.config to code under test above import
+// (but can't do it with the jest mocks that get auto-hoisted)
+// eslint-disable-next-line import/first
+import config from './config';
+
+jest.mock('lib/pad', () => jest.fn(arg => arg));
+
+afterAll(() => {
+    window.guardian.config = prevConfig;
```

Comment:
I don't think this is strictly necessary as each test spec file should run in its own environment. I tested this by removing the `afterAll` here and `console.log`ging in all other test files to ensure the `page` property was not present. 

Also I think `prevConfig` at this point is pointing to `window.guardian.config` anyway, so this is not doing anything.